internal/cluster/clusterworkflow/activity_create_node_pool.go

```diff
@@ -161,6 +167,44 @@ func (a CreateNodePoolActivity) Execute(ctx context.Context, input CreateNodePoo
 			}
 		}
 
+		var amiImageSize int
+		{
+			activityOutput, err := eksworkflow.NewGetAMIImageSizeActivity(
+				a.awsSessionFactory,
+				eksworkflow.NewEC2Factory(),
+			).Execute(
+				ctx,
+				eksworkflow.GetAMIImageSizeActivityInput{
+					EKSActivityInput: commonActivityInput,
+					ImageID:          nodePool.Image,
+				},
+			)
+			if err != nil {
+				return err
+			}
+
+			amiImageSize = activityOutput.AMIImageSize
+		}
+
+		var nodeVolumeSize int
```

Comment:
It's called `defaultVolumeSize` above. I think having the word `node` makes sense in a context which is not node specific, but in this case, I think we can drop it. At the very least, it should be consistent within the same context though.

Reply:
We have a mix of `NodeImage` and `Image` field names (similarly with others such as instance type) in our structs, I tried to follow that pattern, that's why in general there are multiple kinds of names.

In this specific case I agree with you and going to drop the node prefix.

Reply:
[Fixed](https://github.com/banzaicloud/pipeline/compare/0e30f5dbe8ec96b1e168c848aadfd65cfc408226..4a93f8bc4925e894ff035584174682ce66d91d93)